### PR TITLE
Add functions to support more operations on services

### PR DESCRIPTION
**Pull request by Yuusha0**
Add functions on services.py needed for manaservices.

### CODE REVIEW

File: manatools/services.py [hunk 1]
```diff
@@ -142,16 +150,20 @@ def set_service(self, service, enable):
...
+#            if dbus_context.is_authorized('org.freeedesktop.policykit.exec', {'polkit.icon': 'abcd', 'aaaa': 'zzzz'}, interactive=True):
+            #     try:
+            #         chkconf = subprocess.run(['/usr/sbin/chkconfig', "-add" if enable else "--del",
+            #                                   service], env=env, timeout=120, check=True, capture_output=True, text=True)
+            #     except (subprocess.CalledProcessError, subprocess.TimeoutExpired):
+            #         # TODO return an exception to the exterior
+            #         print("chkconfig error when trying to add/delete service", stderr)
+            # else:
+            #     # TODO return an excpetion to the exterior
+            #     print("You are not authorized to perform this action", stderr)
```
Comment (anaselli) [inline manatools/services.py:162]: those comments are needed to recall something or have you forgot to remove them?

**Comment by anaselli** [comment]
I see a lot of merge commits from upstream, maybe when you do a merge you could also try to rebase to avoid those commits, that's not a must of course, but it seems to be easier for me to review :)

**Comment by Conan-Kudo** [comment]
What's the reason for not using pystemd?

**Comment by Yuusha0** [comment]
It's because pystemd does not support interactive authentication. It has been reported upstream, and the developer agreed on adding this function. However, it has not been added to pystemd yet.

**Pull request closed without merging**